Updated WF quickstarts version

vprusa opened the pull request:
Updated used WF quickstarts

vprusa commented:
testPR

jkopriva commented:
@vprusa I suggest also updating server requirement to WildFly 11 (and rebase due to failing build).

vprusa commented:
[Build results](https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/examples.wildfly.it.weekly/271/testReport/) fixing failing tests is in progress - jiras:[1](https://issues.jboss.org/browse/JBDS-4636) [2](https://issues.jboss.org/browse/JBDS-4638) [3](https://issues.jboss.org/browse/JBDS-4637)

vprusa commented:
@jkopriva @rawagner

jkopriva commented:
testPR

vprusa commented:
Updated:
- used WF version
- used WF quickstarts - master
- added checkstyle plugin requirements
- ignore tests that were not necessary [template](https://issues.jboss.org/browse/JBDS-4637) or plan to be removed [spring-petclinic](https://issues.jboss.org/browse/JBDS-4636) 
- ignore documented errors in quickstarts [jts, ejb-security-interceptors & messaging-clustering-singleton](https://issues.jboss.org/browse/JBDS-4638) and for [jaxws-retail](https://github.com/wildfly/quickstart/tree/master/jaxws-retail) necessity to generate source files from WSDL file after quickstart is imported 
- ignore documented errors required usage of Import-Package: org.json.simple & org.json.simple.parser

[Green run](https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/examples.wildfly.it.weekly/279/)

rawagner commented on tests/org.jboss.tools.examples.ui.bot.test/META-INF/MANIFEST.MF:
```diff
@@ -5,11 +5,13 @@ Bundle-SymbolicName: org.jboss.tools.examples.ui.bot.test;singleton:=true
  org.jboss.ide.eclipse.as.reddeer
-Import-Package: org.eclipse.wst.server.ui.internal.view.servers,
- org.eclipse.reddeer.requirements.server
+Import-Package: org.eclipse.reddeer.requirements.server,
```
In this case it is better to use Require-Bundle instead of Import-Package for reddeer and eclipse stuff

jkopriva commented on tests/org.jboss.tools.examples.ui.bot.test/pom.xml:
```diff
@@ -16,82 +16,107 @@
 		<!-- -DimportTestDefinition=${importTestDefinition}</systemProperties> -->
 		<systemProperties>${integrationTestsSystemProperties} -DexamplesLocation=${project.build.directory}/${examplesFolder} -Dmaven.settings.path=${maven.settings.path} -Drd.config=${reddeer.config} -DspecificQuickstarts=${specificQuickstarts} -DdeployOnServer=${deployOnServer}</systemProperties>
-		<quickstartsURLWildFly>https://github.com/wildfly/quickstart/archive/10.x.zip</quickstartsURLWildFly>
+		<quickstartsURLWildFly>https://github.com/wildfly/quickstart/archive/master.zip</quickstartsURLWildFly>
```
@vprusa We should test latest released WildFly with corresponding quickstarts, not from master.

jkopriva commented:
@vprusa Please also update readme

vprusa commented:
README changes will be in EAP updated PR soon.

jkopriva commented:
LGTM +1
